core/test/financial_templates/ExpiringMultiParty.js

```diff
@@ -1,23 +1,24 @@
-const ExpiringMultiParty = artifacts.require("ExpiringMultiParty");
-// Helper Contracts
-const ERC20MintableData = require("@openzeppelin/contracts/build/contracts/ERC20Mintable.json");
-const truffleContract = require("@truffle/contract");
-const ERC20Mintable = truffleContract(ERC20MintableData);
-ERC20Mintable.setProvider(web3.currentProvider);
+// const ExpiringMultiParty = artifacts.require("ExpiringMultiParty");
+// const Finder = artifacts.require()
+// // Helper Contracts
+// const ERC20MintableData = require("@openzeppelin/contracts/build/contracts/ERC20Mintable.json");
+// const truffleContract = require("@truffle/contract");
+// const ERC20Mintable = truffleContract(ERC20MintableData);
+// ERC20Mintable.setProvider(web3.currentProvider);
 
-contract("ExpiringMultiParty", function(accounts) {
-  it("Empty", async function() {
-    const collateralAddress = await ERC20Mintable.new({ from: accounts[0] });
-    const { toWei } = web3.utils;
-    await ExpiringMultiParty.new(
-      true,
-      "1234567890",
-      "1000",
-      collateralAddress.address,
-      { rawValue: toWei("0.1") },
-      { rawValue: toWei("0.1") },
-      { rawValue: toWei("0.1") },
-      1000
-    );
-  });
-});
+// contract("ExpiringMultiParty", function(accounts) {
```

Comment:
Why is this commented out?

Reply:
right now we dont need it in any way. it's just a placeholder. I can remove it or implement the interface fully. open for input here

Reply:
IMO, as long as the ExpiringMultiparty class exists and has a constructor, we should have at least one test that runs that constructor to make sure it's not broken.

Reply:
ok will do.